sdk/search/search/review/search.api.md

```diff
@@ -67,6 +290,34 @@ export interface GetDocumentOptions<Fields> extends OperationOptions {
     selectedFields?: Fields[];
 }
 
+// @public (undocumented)
+export type GetIndexOptions = OperationOptions;
+
+// @public (undocumented)
+export type GetIndexStatisticsOptions = OperationOptions;
+
+// @public
+export interface GetIndexStatisticsResult {
+    readonly documentCount: number;
+    readonly storageSize: number;
+}
+
+// @public
+export interface Index {
+    analyzers?: AnalyzerUnion[];
+    charFilters?: CharFilterUnion[];
+    corsOptions?: CorsOptions;
+    defaultScoringProfile?: string;
+    encryptionKey?: EncryptionKey;
+    eTag?: string;
```

Comment:
Elsewhere we've lowercased this to `etag` (as that seemed to be the predominant practice in JS libs I could find)